src/migrate.rs

```diff
@@ -120,7 +120,7 @@ mod tests {
     #[test]
     fn test_migrate_invalid_version() {
         let mut deps = mock_provenance_dependencies();
-        let current_version: String = "0.4.0".into();
+        let current_version: String = "1.1.0".into();
```

Comment:
Would it be worth setting this to something like `1000.1.0` so that you don't have to bump the test every time the package version is bumped?

Reply:
Good idea